_includes/connect-zbt-1/faq/is-zgp-supported.md

```diff
@@ -0,0 +1 @@
+No. Zigbee Green Power is not supported by {{ productName }}.
```

Comment:
Technically, ZGP is not supported by the ZHA integration, this change makes it sound as if there is a hardware limitation, but there isn't.

So I'd leave it as is (without mentioning the product name), or maybe better, state that ZGP is not supported by the recommended ZHA integration. Afaik Z2M supports ZGP, but I am not sure if the combination with ZBT-1/our firmwares is well tested/works, so I'd not make a statement about that at this point.